### ISSUE TITLE
tsd and typings is dead bro!

### ISSUE THREAD

**Issue by ghost** [tonysneed/generator-tonysneed-vscode-typescript#4]
Hey bro! TSD and typings are dead! https://github.com/Microsoft/TypeScript/issues/9184

Now you need to NPM install `@types/jasmine`

For `npm install typescript@next`

Love yeah!

**Comment by tonysneed** [comment]
Yes, I agree .. I can make the change, or if you like, you can make it and submit a pull request. I'm good either way.
